order/buyer/issue/no-delivery-expected.md

```diff
@@ -14,5 +14,5 @@ When a line has no goods to be delivered, for example a service, fee or text lin
 You can set `No delivery expected` by using the `/order` API resource, by setting the `indicators.noDeliveryExpected` on line level.
 
 {% hint style="info" %}
-Lines having the `noDeliveryExpected` indicator set will never become`Overdue`
+Lines having the `noDeliveryExpected` indicator set will never become `Overdue`.
```

Comment:
This indicator is not implemented

Reply:
@Sjorstradecloud shall I remove `noDeliveryExpected` completely from the API docs and specs?
If so, I will do this in additional PRs